lib/my_home_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_project/search_engine.dart';
+import 'package:flutter_project/styles.dart';
+
+import 'button_section.dart';
+import 'movie.dart';
+import 'movie_details.dart';
+import 'movie_genres.dart';
+import 'movie_header.dart';
+import 'movie_presentation.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key, required this.title});
+
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final Movie movie = Movie.marioBros();
+
+  static const Color scaffoldBackground = Color(0xff1C091C);
+
+  static const double paddingBetween = 30;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: scaffoldBackground,
+      appBar: AppBar(
+        leading: const Icon(
+          Icons.person,
+          color: Colors.white,
+        ),
+        title: const Text(
+          appTitle,
+          style: TextStyle(
+            color: Colors.white54,
+          ),
+        ),
+        backgroundColor: Colors.black45,
+      ),
+      body: SafeArea(
+        child: SingleChildScrollView(
+          physics: const BouncingScrollPhysics(),
+          scrollDirection: Axis.vertical,
+          child: Column(
+            children: [
+              const SearchSection(),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: paddingBetween,
+                ),
+                child: MovieHeader(movie: movie),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: paddingBetween,
+                ),
+                child: MoviePresentation(movie: movie),
+              ),
```

Comment:
We can create a method that receives a child, and returns this padding with that child attached to it

Reply:
Solved